### PR TITLE
fix: provide way to do dispatch.include for Ajax ResourceRequest on views

### PR DESCRIPTION
**Pull request by jgribonvald**
Referenced to spring-webmvc change that do a forward for resource request whereas include was used before.
After spring 3.1.2 we get this issue: https://github.com/spring-projects/spring-framework/issues/15417 which is caused by this commit: https://github.com/spring-projects/spring-framework/commit/b17d53ffae3b464ce1e550e3a4bc1c3073087323 -  `DispatcherPortlet uses a forward for rendering a view as resource response Issue: SPR-9876`

### CODE REVIEW

File: src/main/java/org/esupportail/portlet/filemanager/portlet/CustomDispatcherPortlet.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+     * @param request current portlet render/resource request
+     * @param response current portlet render/resource response
+     * @throws Exception if there's a problem performing the dispatch
+     */
+    @Override
+    protected void doDispatch(PortletRequestDispatcher dispatcher, PortletRequest request, MimeResponse response)
+            throws Exception {
+        // Comment fix code as getting this problem https://github.com/spring-projects/spring-framework/issues/15417
+        // TODO need to rebuild the portlet => WebComponent + API to fix
+//        if (PortletRequest.RESOURCE_PHASE.equals(request.getAttribute(PortletRequest.LIFECYCLE_PHASE))) {
```
Comment (ChristianMurphy) [inline src/main/java/org/esupportail/portlet/filemanager/portlet/CustomDispatcherPortlet.java:25]: Is this commented code going to be needed, could it be removed?

**Review by ChristianMurphy — COMMENTED** [review]
Thanks @jgribonvald,
A question below

File: src/main/java/org/esupportail/portlet/filemanager/portlet/CustomDispatcherPortlet.java (hunk 1 above)
Comment (jgribonvald) [reply src/main/java/org/esupportail/portlet/filemanager/portlet/CustomDispatcherPortlet.java:25]: The commented code is the original one to remember what was there before. 
In my mind rebuilding the API and the frontend is needed and so this code will be removed. So it's in waiting better and to "document" what is done here.

**Comment by jgribonvald** [comment]
> Skimming the code it LGTM. I'm away from my computer though, and haven't tested this on a running portal instance.

I've some other code update coming, I need to merge it before doing a PR with several commits, so it's like you want ;)

**Pull request merged**